src/styles/table.scss

```diff
@@ -152,16 +152,6 @@
   }
 }
 
-@container (min-width: 1400px) {
```

Comment:
We cannot remove this, it breaks other tables so the table outter paddings don't match header paddings:

![image](https://github.com/Layer-Fi/layer-react/assets/11715931/18899d85-8147-4a06-854b-5f80c3f8cd7a)